input/cql/Status.cql

```diff
@@ -124,6 +144,11 @@ define fluent function isObservationBMI(Obs List<"USCoreBMIProfile">):
     where O.status in { 'final', 'amended', 'corrected' }
 
 //Symptom
-define fluent function isSymptom(Obs List<Choice<"SimpleObservation", "ObservationScreeningAssessment">>):
```

Comment:
According to the [mapping](https://hl7.org/fhir/us/qicore/STU6/qdm-to-qicore.html#symptom), this should be [QI-Core Simple Observation](https://hl7.org/fhir/us/qicore/STU6/StructureDefinition-qicore-simple-observation.html)

Reply:
Can't symptom be also considered in an ObservationClinicalResult and ObservationScreeningAssessment?